evaluation/swe_bench/swe_env_box.py

```diff
@@ -73,6 +73,10 @@ def get_box_for_instance(
             )
         config.workspace_base = workspace_mount_path
         config.workspace_mount_path = workspace_mount_path
+
+        # linting after editing boosts benchmark scores
+        config.enable_auto_lint = True
```

Comment:
Maybe you can update the `config.toml` example in the `README.md` of `evaluation/swe_benc` - just for safe?

Reply:
make sense. Updated.